<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>model_fitting_plugin</class>
 <widget class="QWidget" name="model_fitting_plugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layout_vertical">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add Model</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="combo_box_models">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_select_model">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_current_models">
     <property name="title">
      <string>Current Models</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeWidget" name="tree_widget_current_models">
        <property name="styleSheet">
         <string notr="true">QTreeWidget::indicator:checked {
    image: url(:/icons/lock.svg);
}

QCheckBox::indicator:checked:hover {
    image: url(:/icons/lock.svg);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/icons/lock.svg);
}

QTreeWidget::indicator:unchecked {
    image: url(:/icons/unlock.svg);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/icons/unlock.svg);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/icons/unlock.svg);
}</string>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Parameter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>State</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="button_save_model">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Save the model as a yaml file</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/Save-48.png</normaloff>:/icons/Save-48.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="button_load_model">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Open a previously saved model</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/Open Folder-48.png</normaloff>:/icons/Open Folder-48.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="button_export_model">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Export the model as a python script</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/Export-48.png</normaloff>:/icons/Export-48.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="button_remove_model">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete selected sub model</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/Delete-48.png</normaloff>:/icons/Delete-48.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="group_box_arithmetic">
        <property name="title">
         <string>Arithmetic</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="line_edit_model_arithmetic"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_fitting">
     <property name="title">
      <string>Fitting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="combo_box_fitting"/>
      </item>
      <item>
       <widget class="QCheckBox" name="check_box_advanced_settings">
        <property name="styleSheet">
         <string notr="true">.QCheckBox::indicator:unchecked {
    image: url(:/icons/icons8-Forward-96.png);
	height: 16px;
	width: 16px;
}

.QCheckBox::indicator:unchecked:pressed {
    image: url(:/icons/icons8-Forward-96.png);
	height: 16px;
	width: 16px;
}

.QCheckBox::indicator:checked {
    image: url(:/icons/icons8-Expand Arrow-96.png);
	height: 20px;
	width: 20px;
}

.QCheckBox::indicator:checked:pressed {
    image: url(:/icons/icons8-Expand Arrow-96.png);
	height: 20px;
	width: 20px;
}</string>
        </property>
        <property name="text">
         <string>Advanced settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_box_advanced_settings">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="max_iterations_label">
           <property name="text">
            <string>Max Iterations</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="max_iterations_line_edit">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="relative_error_label">
           <property name="text">
            <string>Relative Error</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="relative_error_line_edit">
           <property name="text">
            <string>1e-07</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="epsilon_label">
           <property name="text">
            <string>Epsilon</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="epsilon_line_edit">
           <property name="text">
            <string>1.49e-08</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_perform_fit">
        <property name="text">
         <string>Perform Fit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
